### DISCUSSION TITLE
default values

### DISCUSSION THREAD

**Discussion by paulwvnjohi** [mskocik/svelecte#89] [Q&A] [answered]
is there a way to add default values to the select? 
it would also be nice to hide the drop down if i focus pragmatically unit keydown is detected- i could open  a PR

**Comment by paulwvnjohi** [comment, marked as answer]
nvm binding `value` works 🧐

**Reply by mskocik** [reply]
It's explained in [minimal-example section](https://github.com/mskocik/svelecte#minimalistic-example). Especially in _"💭 Note about `value` and `readSelection` property"_.

**Comment by mskocik** [comment]
> it would also be nice to hide the drop down if i focus pragmatically unit keydown is detected

I am not sure, if I want support this behavior out of the box 🤔 But basically you can achieve this by dispatching `blur` event on input. Take a look here: [REPL](https://svelte.dev/repl/508f2c1f5f9141d5ae646ac9a13a4ae1?version=3.46.4)
